Source/GUI/dvrescue/dvrescue/dvrescue.pro

```diff
@@ -1,6 +1,6 @@
 QT += qml quick widgets
 
-CONFIG += c++11
+CONFIG += c++17
```

Comment:
This removes the support of CentOS 7 and Debian 9. They are not the latest though, and we don't have much users on theses distro and platfoms, so could be OK, cc @dericed.
IIRC for macOS and Windows it is fine (a build on a newer platform version is compatible with older versions).

Reply:
from the other side Qt 6 needs C++17 (if decision will be made to support it one day)